packages/lib/product/service.ts

```diff
@@ -163,6 +164,46 @@ export const getProductByEnvironmentId = reactCache(
     )()
 );
 
+export const getProductForEnvironmentState = reactCache(
```

Comment:
same for this, please move this to `apps/web/app/api/v1/client/[environmentId]/environment/lib/product.ts`